From: =?utf-8?q?Matthias_Bl=C3=BCmel?= <blaimi@blaimi.de>
Date: Wed, 12 Jun 2019 22:15:44 +0200
Subject: use generated autoloader-file for unittests

The testfiles are not included in binary which makes it necessary to use a different autoloader-file with additional classes for testing.

Forwarded: not-needed
---
 test/bootstrap-dist.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/bootstrap-dist.php b/test/bootstrap-dist.php
index 1c4d6ea..45c9284 100644
--- a/test/bootstrap-dist.php
+++ b/test/bootstrap-dist.php
@@ -56,7 +56,7 @@ if (PHP_SAPI == 'cli') {
 }
 
 require_once 'libraries/vendor_config.php';
-require_once AUTOLOAD_FILE;
+require_once __DIR__ . '/autoload.php';
 PhpMyAdmin\MoTranslator\Loader::loadFunctions();
 $CFG = new PhpMyAdmin\Config();
 // Initialize PMA_VERSION variable
